source/plugins/json/index.md

```diff
@@ -367,6 +367,19 @@ given result add encoding param as below:
 </result>
 ```
 
+### Customizing the output
+
+Since `2.5.14`, you can implement the `org.apache.struts2.json.JSONWriter` interface to customize the generated json response.
```

Comment:
It is better to use a full product name, i.e. `Since Struts 2.5.14 ...`

Reply:
Thank you; fixed :)